tests/network/vmi_multus.go

```diff
@@ -572,6 +573,82 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 				Expect(testErr.ErrStatus.Reason).To(BeEquivalentTo("Invalid"))
 			})
 		})
+
+		Context("Security.", func() {
+			BeforeEach(func() {
+				const (
+					bridge11CNIType       = "cnv-bridge"
+					bridge11Name          = "br11"
+					bridge11MACSpoofCheck = true
+				)
+
+				Expect(createBridgeNetworkAttachmentDefinition(util.NamespaceTestDefault,
+					linuxBridgeWithMACSpoofCheckNetwork,
+					bridge11CNIType,
+					bridge11Name,
+					0,
+					"",
+					bridge11MACSpoofCheck)).To(Succeed())
+			})
+
+			It("Should allow outbound communication from VM under test - only if original MAC address is unchanged", func() {
+				const (
+					vmUnderTestIPAddress = "10.2.1.1"
+					targetVMIPAddress    = "10.2.1.2"
+					bridgeSubnetMask     = "/24"
+				)
+
+				initialMacAddress, err := tests.GenerateRandomMac()
+				Expect(err).NotTo(HaveOccurred())
+				initialMacAddressStr := initialMacAddress.String()
+
+				spoofedMacAddress, err := tests.GenerateRandomMac()
+				Expect(err).NotTo(HaveOccurred())
+				spoofedMacAddressStr := spoofedMacAddress.String()
+
+				// TODO - change to libvmi.InterfaceDeviceWithBridgeBinding(linuxBridgeWithMACSpoofCheckNetwork)
+				// after PR https://github.com/kubevirt/kubevirt/pull/6797 is merged
+				linuxBridgeInterfaceWithMACSpoofCheck := v1.Interface{
+					Name: linuxBridgeWithMACSpoofCheckNetwork,
+					InterfaceBindingMethod: v1.InterfaceBindingMethod{
+						Bridge: &v1.InterfaceBridge{},
+					},
+				}
+
+				By("Creating a VM with custom MAC address on its Linux bridge CNI interface.")
+				linuxBridgeInterfaceWithCustomMac := linuxBridgeInterfaceWithMACSpoofCheck
+				libvmi.InterfaceWithMac(&linuxBridgeInterfaceWithCustomMac, initialMacAddressStr)
+
+				vmiUnderTest := libvmi.NewTestToolingFedora(
+					libvmi.WithInterface(linuxBridgeInterfaceWithCustomMac),
+					libvmi.WithNetwork(libvmi.MultusNetwork(linuxBridgeWithMACSpoofCheckNetwork)),
+					libvmi.WithCloudInitNoCloudNetworkData(cloudInitNetworkDataWithStaticIPsByMac(linuxBridgeInterfaceWithCustomMac.Name, linuxBridgeInterfaceWithCustomMac.MacAddress, vmUnderTestIPAddress+bridgeSubnetMask), false))
+				vmiUnderTest = tests.CreateVmiOnNode(vmiUnderTest, nodes.Items[0].Name)
+
+				By("Creating a target VM with Linux bridge CNI network interface and default MAC address.")
+				targetVmi := libvmi.NewTestToolingFedora(
+					libvmi.WithInterface(linuxBridgeInterfaceWithMACSpoofCheck),
+					libvmi.WithNetwork(libvmi.MultusNetwork(linuxBridgeWithMACSpoofCheckNetwork)),
+					libvmi.WithCloudInitNoCloudNetworkData(cloudInitNetworkDataWithStaticIPsByDevice("eth0", targetVMIPAddress+bridgeSubnetMask), false))
+				targetVmi = tests.CreateVmiOnNode(targetVmi, nodes.Items[0].Name)
+
+				vmiUnderTest = tests.WaitUntilVMIReady(vmiUnderTest, console.LoginToFedora)
+				tests.WaitUntilVMIReady(targetVmi, console.LoginToFedora)
+
+				By("Pinging target VM from VM under test - before MAC spoof.")
```

Comment:
I think you mean "before MAC is changed", or just "with original MAC".

You could also just do:
`Expect(libnet.PingFromVMConsole(vmiUnderTest, targetVMIPAddress)).To(Succeed(), "ping target IP with original MAC should succeed")` and drop the `By`.

Reply:
Done